### PR TITLE
Centauri-updates

### PR DESCRIPTION
**Pull request by JafarAz**
- Updated Centauri content with the latest architecture, diagrams, content and more. 

- Aped a few more miscellaneous items.

- Will update sidebars.js in another PR once this PR is approved

### CODE REVIEW

File: docs/docs/codespaces/getting-started.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ You check this by going to [github.com/orgs/ComposableFi/people](https://github.
 ![GitHub Organization Check](./github-organization-check.png)
 *Search for your own GitHub username*
 
-**If you are not yet part of the ComposableFi GitHub organization, send a message to `@cor` in the composable Slack 
-channel.**
+**If you are not yet part of the ComposableFi GitHub organization, create a ticket with R2D2 in the composable
```
Comment (blasrodri) [inline docs/docs/codespaces/getting-started.md:20]: Does this exist?

File: docs/docs/products/centauri/Dotsama-ibc.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# DotSama IBC
+_IBC beyond the Cosmos_
+## Kusama ⬌ Polkadot Implementation
+
+The first implementation of Centauri will be between [Kusama](https://kusama.network/) and [Polkadot](https://polkadot.network/) via Picasso and Composable. This is a significant development because currently there is no bridge connecting these two relay chains. This also marks the **first implementation of IBC outside of the Cosmos ecosystem**.
```
Comment (blasrodri) [inline docs/docs/products/centauri/Dotsama-ibc.md:5]: Present

File: docs/docs/products/centauri/cosmos11-BEEFY-COSMOS-IBC-light-client.md [hunk 3]
```diff
@@ -67,51 +52,23 @@ pub struct BEEFYLightClient{
 ```
 
 
-Composable is performing a total of 8 PRs to core BEEFY subsystems in both the runtime and Substrate client, pending 
-further review by the Substrate bridges team. Some are listed below:
+Composable is performing a total of 8 PRs to core BEEFY subsystems in both the runtime and Substrate client, pending further review by the Substrate bridges team. Some are listed below:
```
Comment (blasrodri) [inline docs/docs/products/centauri/cosmos11-BEEFY-COSMOS-IBC-light-client.md:55]: I would remove this @0xBrainjar2

**Review by blasrodri — DISMISSED** [review]

File: docs/docs/codespaces/getting-started.md (hunk 1 above)
Comment (JafarAz) [reply docs/docs/codespaces/getting-started.md:20]: yessir still exists

File: docs/docs/products/centauri/cosmos11-BEEFY-COSMOS-IBC-light-client.md (hunk 3 above)
Comment (0xBrainjar2) [reply docs/docs/products/centauri/cosmos11-BEEFY-COSMOS-IBC-light-client.md:55]: agree

File: docs/docs/products/centauri-overview.md [hunk 4]
```diff
@@ -1,39 +1,35 @@
...
-
-![overview_centauri_progress](./overview-centauri-progress.png)
-
-
-To create the proof of concept (PoC) of Centauri, we are working with a variety of projects in the Cosmos ecosystem. 
-Then, we will hold a larger-scale launch of the bridge, serving as our fundamental IBC infrastructure for trustless 
-bridging.
+_Trustless IBC transfer protocol_
+
+Centauri is a decentralized transfer protocol that will facilitate trustless cross-ecosystem communication among various blockchains, including Polkadot, Kusama, Ethereum, NEAR, and Cosmos, using the IBC Protocol.
```
Comment (0xBrainjar2) [inline docs/docs/products/centauri-overview.md:5]: is an extension of the IBc protocol

File: docs/docs/products/centauri-overview.md [hunk 5]
```diff
@@ -1,39 +1,35 @@
...
+
+:::tip Centauri Is Live
+
+Use Centauri now to transfer assets between Polkadot and Kusama. Centauri is in advanced testnet stages to its Cosmos connection while we are actively working on implementing IBC to other blockchains.
+
+:::
+
+Centauri leverages and expands upon the existing Inter-Blockchain Communication Protocol (IBC) beyond Cosmos. The IBC protocol previously allowed for trustless bridging between Cosmos SDK chains; however, we are amongst the first to extend IBC to other ecosystems.
```
Comment (0xBrainjar2) [inline docs/docs/products/centauri-overview.md:13]: are the first

File: docs/docs/products/centauri-overview.md [hunk 6]
```diff
@@ -1,39 +1,35 @@
...
+
+Use Centauri now to transfer assets between Polkadot and Kusama. Centauri is in advanced testnet stages to its Cosmos connection while we are actively working on implementing IBC to other blockchains.
+
+:::
+
+Centauri leverages and expands upon the existing Inter-Blockchain Communication Protocol (IBC) beyond Cosmos. The IBC protocol previously allowed for trustless bridging between Cosmos SDK chains; however, we are amongst the first to extend IBC to other ecosystems. 
+
+Similar to any IBC connection between two chains, Centauri supports asset transfers (fungible tokens, non-fungible tokens), generic message passing, cross-chain contract calls, cross-chain fee payments, interchain collateralization and more in a trustless manner. The trustless condition of Centauri is due to the fact it is:
+
+- built upon light clients that communicate with each other
```
Comment (0xBrainjar2) [inline docs/docs/products/centauri-overview.md:17]: light client verification

**Review by 0xBrainjar2 — CHANGES_REQUESTED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**